corehq/toggles/__init__.py

```diff
@@ -999,19 +999,6 @@ def _ensure_valid_randomness(randomness):
     """
 )
 
-CASE_SEARCH_INDEXED_METADATA = StaticToggle(
```

Comment:
BHA has been using this without incident for a while.  We only built it behind a flag out of abundance of caution anyways.